system_metrics_collector/src/system_metrics_collector/linux_process_memory_measurement_node.hpp

```diff
@@ -19,8 +19,9 @@
 #include <cmath>
 #include <string>
 
-#include "../../src/system_metrics_collector/periodic_measurement_node.hpp"
-#include "../../src/system_metrics_collector/utilities.hpp"
+#include "./visibility_control.h"
```

Comment:
Couldn't you just use ` #include "visibility_control.h"` ?

Reply:
cpplint will complain about
```
Include the directory when naming .h files  [build/include] [4]
```

It doesn't complain like this about `.hpp` files, but I don't think this header should be a `.hpp` file.